packages/itwinui-css/src/style/global.scss

```diff
@@ -35,31 +35,38 @@ html {
   }
 }
 
-.iui-body {
-  background-color: var(--iui-color-background-2);
+:where(.iui-root) {
   color: var(--iui-text-color);
   font-size: $iui-font-size;
   line-height: $iui-line-height;
 
-  @include iui-scrollbar;
   @include iui-font-family;
 
-  &.iui-no-webfonts {
-    font-family: $iui-sans-fallback;
+  // reset margin and apply background only for <body> element
+  @at-root body#{&} {
+    margin: 0;
+    background-color: var(--iui-color-background-2);
   }
-}
-
-[class*='iui-'],
-[class*='iui-'] * {
-  // Text highlight
-  @include iui-text-selection;
 
-  // Scrollbar
-  @include iui-scrollbar;
+  // border-box on all iui- elements and their descendants under iui-root, excluding any nested iui-root elements
+  :where([class*='iui-']) {
+    &:where(:not(.iui-root)),
+    * {
+      &,
+      &::before,
+      &::after {
+        box-sizing: border-box;
+      }
+    }
+  }
 }
 
-[class*=iui-]:where(:not(.iui-body)),
-[class*=iui-]:where(:not(.iui-body))::before,
-[class*=iui-]:where(:not(.iui-body))::after {
```

Comment:
Are these lines replaced by the new lines 52-62? Did you replace it with the new code since the new code is more efficient? Just wondering, because the old way is lesser lines of code.

Reply:
The old way was doing something entirely different and it was not correctly targeting nested elements https://github.com/iTwin/iTwinUI/pull/684#discussion_r914195877